pilot/pkg/proxy/envoy/discovery.go

```diff
@@ -82,6 +86,20 @@ var (
 			Name:      "errors",
 			Help:      "Counter of errors encountered during a given method call within Pilot",
 		}, []string{metricLabelMethod, metricBuildVersion})
+	webhookCallCounter = prometheus.NewCounterVec(
```

Comment:
General question about Istio component metrics: are they considered a stable API, or can we remove/rename without expectation of backwards compatibility? cc @nmittler @douglas-reid

Reply:
They definitely should not be part of our API today. At 1.0 we can start having that discussion.